### PR TITLE
Add v5.5.5 upgrade

### PR DESCRIPTION
**Pull request by Kbhat1**
## Describe your changes and provide context
- Add v5.5.5 upgrade

## Testing performed to validate your change
- Tested on local chain

### CODE REVIEW

**Review by udpatil — COMMENTED** [review]
Can we update the changelog too please?

**Review by philipsu522 — APPROVED** [review]
+1 for changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,10 +26,22 @@ Types of changes (Stanzas):
 Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
-# Changelog
+# Changelog## v5.5.5
```
Comment (Sulthanmh) [inline CHANGELOG.md:29]: #

Comment (Sulthanmh) [reply CHANGELOG.md:29]: #

**Pull request merged**